Tools/BuildAndPackage.ps1

```diff
@@ -75,17 +120,29 @@ Write-Host -ForegroundColor Cyan "*** HACK: Creating GoogleCloudPowerShell modul
 
 New-Item -ItemType Directory $packageDir
 New-Item -ItemType Directory $gcpsDir
-Move-Item (Join-Path $debugDir "GoogleCloudPowerShell.psd1") "$gcpsDir\GoogleCloudPowerShell.psd1"
+
+# Add the version folder in front of Google.PowerShell.dll and GoogleCloudPlatform.Format.ps1xml.
+$gCloudPowerShellManifest = Import-PowerShellDataFile (Join-Path $debugDir "GoogleCloudPowerShell.psd1")
+Remove-PrivateData $gCloudPowerShellManifest
+$gCloudPowerShellManifest.RootModule = $gCloudPowerShellManifest.RootModule -replace "Google.PowerShell.dll", `
+                                                                                   "$normalizedVersion\Google.PowerShell.dll"
+$gCloudPowerShellManifest.FormatsToProcess = $gCloudPowerShellManifest.FormatsToProcess -replace "GoogleCloudPlatform.Format.ps1xml", `
+                                                                                               "$normalizedVersion\GoogleCloudPlatform.Format.ps1xml"
+$gCloudPowerShellManifest.CmdletsToExport = $cmdletsList
+New-ModuleManifest -Path "$gcpsDir\GoogleCloudPowerShell.psd1" @gCloudPowerShellManifest
 
 # Package the bits. Requires setting up the right directory structure.
 Write-Host -ForegroundColor Cyan "*** Packaging the bits ***"
 
 New-Item -ItemType Directory $powerShellDir
 Copy-Item -Recurse $debugDir $powerShellDir
 
-# The binaries in a folder named "Debug". Rename that to "GooglePowerShell".
+# The binaries are in a folder named "Debug". Move them to GoogleCloud\$normalizedVersion folder.
```

Comment:
Might be outside the scope of this review but does this mean they are being built with debug instead of as release?